File: batches_csharp/batch_02_variable_renaming/ref.cs
using System;

class SumOfSquares {
    static int SumSquares(int[] numbers) {
        int total = 0;
        foreach (int num in numbers) total += num * num;
        return total;
    }

    static int ProcessList(int[] data) => SumSquares(data);

    static void Main() {
        int[] myList = {1, 2, 3, 4, 5};
        Console.WriteLine(ProcessList(myList));
    }
}

File: batches_java/batch_02_variable_renaming/ref.java
public class SumOfSquares {
    public static int sumOfSquares(int[] numbers) {
        int total = 0;
        for (int num : numbers) {
            total += num * num;
        }
        return total;
    }

    public static int processList(int[] data) {
        return sumOfSquares(data);
    }

    public static void main(String[] args) {
        int[] myList = {1, 2, 3, 4, 5};
        System.out.println(processList(myList));
    }
}
